app/assets/stylesheets/forever_style_guide/globals/_colors.scss

```diff
@@ -4,15 +4,23 @@
 ---------------------------------------------------------------------------- */
 
 // Core color declarations, this list is intended to be a limited set of unique branded colors
-$color-forever_green:   colorify('green', #7ac143) !default;
-$color-forever_blue:    colorify('blue', #00a4e4) !default;
-$color-forever_purple:  colorify('purple', #731472) !default;
+$color-forever_green:   colorify('green', #7dc142) !default;
+$color-forever_spare:   colorify('spare', #7ec24d) !default;
```

Comment:
we have one spare color in that spectrum for future use, I didn't know what to call it so 'spare' seemed appropriate for now.